R/utils-hub_tasks_config.R

```diff
@@ -19,3 +19,24 @@ filter_task_groups_to_target <- function(task_groups, target_id) {
 
   return(task_groups)
 }
+
+
+#' Get a character vector of all output types across all task groups
+#' @noRd
+get_output_types <- function(task_groups) {
+  output_types <- purrr::map(
+    task_groups,
+    function(task_group) names(task_group[["output_type"]])
+  ) |>
+    unlist() |>
+    unique()
+  return(output_types)
+}
+
+
+#' Get a boolean indicating whether the target is ordinal
+#' @noRd
+get_target_is_ordinal <- function(task_groups_w_target) {
+  target_type <- task_groups_w_target[[1]]$target_metadata[[1]]$target_type
```

Comment:
Double-checking here: we are expecting one and only one target, right? Because we reduce the task groups to the ones with the expected target, there's no sense in checking all of them.

Reply:
That's right -- I added a comment clarifying this in f823d8e